op-mode-definitions/show-arp.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="show">
+    <children>
+    <node name="arp">
+      <properties>
+        <help>Show Address Resolution Protocol (ARP) information</help>
+      </properties>
+      <command>/usr/sbin/arp -e -n</command>
+    </node>
+    <tagNode name="arp">
```

Comment:
This will not work in the new system I think. The commands should be "run show arp" and "run show arp interface", where interface is a tag node.

Reply:
you mean, "show arp" as a node, and move the tag named "show arp" to "show arp interface" ?  
I did this "trick" to keep the original syntax, but if syntax change is accepted its easy to fix.

this trick is done in more places, so fixing this means changeing syntax on every command that does this.

Reply:
There aren't _too_ many commands that use this arrangement I think. I also think this arrangement was always a bad idea. :)

Reply:
👍 I'll update syntax on all commands that does this :)

Reply:
syntax updated